function10.js

```diff
@@ -0,0 +1,14 @@
+
+
+function filtered(board) {
+    for(var i=0; i <= board.length();i++){
```

Comment:
[.length is a property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/length), not a method. You should not use parentheses.